cmake/Modules/SetGPUArchs.cmake

```diff
@@ -0,0 +1,50 @@
+# =============================================================================
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(GPU_ARCHS
```

Comment:
The fact that this is now in this module file makes it harder to know that this is an available option. Maybe there's some way to see what options are available in a cmake file, but I just go look at the `CMakeLists.txt` directly.

Reply:
I have the intention of refactoring this out entirely and just using `CMAKE_CUDA_ARCHITECTURES` which is the standard CMake variable for controlling this

Reply:
Refactored this out to just use `CMAKE_CUDA_ARCHITECTURES`